// ng-http-loader/esm2015/public_api.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
/*
 * Public API Surface of ng-http-loader
 */
export { SkChasingDotsComponent } from './lib/components/sk-chasing-dots/sk-chasing-dots.component';
export { SkCubeGridComponent } from './lib/components/sk-cube-grid/sk-cube-grid.component';
export { SkDoubleBounceComponent } from './lib/components/sk-double-bounce/sk-double-bounce.component';
export { SkRotatingPlaneComponent } from './lib/components/sk-rotating-plane/sk-rotating-plane.component';
export { SkSpinnerPulseComponent } from './lib/components/sk-spinner-pulse/sk-spinner-pulse.component';
export { SkThreeBounceComponent } from './lib/components/sk-three-bounce/sk-three-bounce.component';
export { SkWanderingCubesComponent } from './lib/components/sk-wandering-cubes/sk-wandering-cubes.component';
export { SkWaveComponent } from './lib/components/sk-wave/sk-wave.component';
export { NgHttpLoaderComponent } from './lib/components/ng-http-loader.component';
export { AbstractLoader } from './lib/components/abstract.loader.component';
export { PendingRequestsInterceptor, PendingRequestsInterceptorProvider } from './lib/services/pending-requests-interceptor.service';
export { SpinnerVisibilityService } from './lib/services/spinner-visibility.service';
export { NgHttpLoaderModule } from './lib/ng-http-loader.module';
export { Spinkit, SPINKIT_COMPONENTS } from './lib/spinkits';
